build/Common.prod.props

```diff
@@ -43,6 +43,8 @@
   </Target>
 
   <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <DefaultTargetFrameworks>net6.0;netstandard2.0;net462</DefaultTargetFrameworks>
```

Comment:
It would be good to add a comment listing the only exceptions to this:
- OpenTelemetry.Instrumentation.AspNetCore
- OpenTelemetry.Exporter.Prometheus.AspNetCore